Assets/VRTK/Prefabs/SnapDropZone/VRTK_SnapDropZone.cs

```diff
@@ -154,7 +152,7 @@ public virtual void OnObjectSnappedToDropZone(SnapDropZoneEventArgs e)
 
         public virtual void OnObjectUnsnappedFromDropZone(SnapDropZoneEventArgs e)
         {
-            UnsnapObject();
+            snapAttachMechanicScript.UnsnapObject();
```

Comment:
need to check that `snapAttachMechanicScript` is not null before performing any actions on it, in case it's been set to null at runtime, to prevent exception errors.